src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -1941,7 +1941,7 @@ class ShenandoahConcurrentWeakRootsEvacUpdateTask : public AbstractGangTask {
   ShenandoahVMWeakRoots<true /*concurrent*/> _vm_roots;
 
   // Roots related to concurrent class unloading
-  ShenandoahClassLoaderDataRoots<true /* concurrent */, false /* single thread*/>
+  ShenandoahClassLoaderDataRoots<true /* concurrent */, true /* single thread*/>
```

Comment:
Should the first input be false?

Reply:
No. It runs concurrently, so will need to acquire ClassLoaderGraph_lock.